src/main/java/com/josdem/vetlog/model/PetImage.java

```diff
@@ -14,26 +14,35 @@
 limitations under the License.
  */
 
-package com.josdem.vetlog.model;
-
-import static jakarta.persistence.GenerationType.IDENTITY;
-
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import lombok.Getter;
-import lombok.Setter;
-
-@Entity
-@Getter
-@Setter
-public class PetImage {
-
-    @Id
-    @GeneratedValue(strategy = IDENTITY)
-    private Long id;
-
-    @Column(nullable = false)
-    private String uuid;
-}
+ package com.josdem.vetlog.model;
+
+ import static jakarta.persistence.GenerationType.IDENTITY;
+ 
+ import jakarta.persistence.Column;
+ import jakarta.persistence.Entity;
+ import jakarta.persistence.FetchType;
+ import jakarta.persistence.GeneratedValue;
+ import jakarta.persistence.Id;
+ import jakarta.persistence.JoinColumn;
+ import jakarta.persistence.ManyToOne;
+ import lombok.Getter;
+ import lombok.Setter;
+ 
+ @Entity
+ @Getter
+ @Setter
+ public class PetImage {
+ 
+     @Id
+     @GeneratedValue(strategy = IDENTITY)
+     private Long id;
+ 
+     @Column(nullable = false)
+     private String uuid;
+ 
+     // Relationship mapping to Pet
+     @ManyToOne(fetch = FetchType.LAZY)
+     @JoinColumn(name = "pet_id", nullable = false)
+     private Pet pet;
```

Comment:
Why are you adding a image relationship? This task has nothing to do with images.